### PR TITLE
DEVOPS-930

### PR DESCRIPTION
**Pull request by gshel**
- Update sphinx implementation
- Implement Github Pages
- Implement VERSION SSoT; Create MANIFEST.ini
- Split abstract and concrete dependencies between requirements.txt and setup.py
- Update README.md
- Include extra_requires for docs generation in setup.py

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -8,15 +10,27 @@
...
+    extras_require={
+        "docs": [
+            "sphinx",
+            "sphinx_click",
+            "sphinx_rtd_theme"
+        ]
+    },
     entry_points="""
         [console_scripts]
         harvestcli=hreporting.cli:main
```
Comment (gshel) [inline setup.py:36]: This needs to be fixed. It doesn't actually appear that we have this repo working as a click application.

File: setup.py [hunk 2]
```diff
@@ -8,15 +10,27 @@
     long_description=open("README.md").read(),
     long_description_content_type="text/markdown",
     packages=find_packages(exclude=["contrib", "docs", "tests"]),
-    version="0.1.1",
+    version=open(os.path.join(".", "VERSION")).read().strip(),
```
Comment (rmeyer-taos) [inline setup.py:13]: :+1:

**Review by rmeyer-taos — COMMENTED** [review]
So far so good!

File: setup.py (hunk 1 above)
Comment (gshel) [reply setup.py]: Going to remove all mentions of click from file

**Review by amcchesney-taos — APPROVED** [review]

**Review by ddavisson — COMMENTED** [review]
Looks good.

**Pull request merged**